docker/wiremock/stubs/mappings/sdsGetStructuredRecord.json

```diff
@@ -5,16 +5,13 @@
 		"urlPath": "/Endpoint",
 		"queryParameters": {
 			"organization": {
-				"equalTo": "https://fhir.nhs.uk/Id/ods-organization-code|GP0001"
+				"matches": "https:\/\/fhir[.]nhs[.]uk\/Id\/ods-organization-code[|].*"
 			},
 			"identifier": {
 				"equalTo": "https://fhir.nhs.uk/Id/nhsServiceInteractionId|urn:nhs:names:services:gpconnect:fhir:operation:gpc.getstructuredrecord-1"
 			}
 		},
 		"headers": {
-			"X-Correlation-Id": {
```

Comment:
Why remove this? SDS API required a UUID

Reply:
I confirmed this with @bartek-sarul, sds does not require this.

Reply:
I see the comment below, I think out wiremock should require it. Our adaptor must always provide this header for tracing.

Reply:
GPC Consumer does not provide this id when performing a request. It should be implemented there before adding it back to wiremock.

Reply:
Added a story for this https://gpitbjss.atlassian.net/browse/NIAD-1225

Reply:
Believe this will now needed to added back in